### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by FominSergiy**
made changes to how init state is generated and moved util functions to a separate file

### CODE REVIEW

File: src/reducer.js [hunk 1]
```diff
@@ -1,55 +1,29 @@
 import { createStore, combineReducers } from 'redux';
+import { getBoardInitialState } from './utils.js';
+import { BOARD_SIZE } from './constants.js';
 
-const boardState = [];
+const boardState = getBoardInitialState(BOARD_SIZE);
```
Comment (karlowski) [inline src/reducer.js:5]: very nice solution instead of loop!

**Review by karlowski — APPROVED** [review]
very nice!

File: src/reducer.js (hunk 1 above)
Comment (FominSergiy) [reply src/reducer.js:5]: Thanks! It's hard to get out of the loop mindset though :(

**Review by FominSergiy — COMMENTED** [review]
Merging with master

**Pull request merged**